### ISSUE TITLE
New career paths

### ISSUE THREAD

**Issue by vinfamy** [vinfamy/LifePlay#76]

**Comment by DRSSX4756** [comment]
I experimented a tiny bit with developing a new career (masseuse at a gym) before I got sidetracked with an NV project. 
The related functions seem to work: mobeJobs, loseJob, setBoss/Salary, moveTo(work) and the like. 

It is a bit difficult though to properly trigger scenes for it, ie being able to look up that you do in fact have that job. Using the 'work' parameter for the WHERE condition makes it trigger if you have the office job instead. Using the 'sports_centre' param makes them trigger if you haven't taken the job yet. I figured I'd park the intel in a global but the OTHER condition doesn't seem to able to check it directly there, unless I did it wrong - something tells me though that you're  meant to first pass a global to a local var and check it that way.

I ended up having to use a stat, which isn't ideal b/c players can tamper with it, bypassing necessary code.
So I was wondering if perhaps a function could be made that stores a string of your job title (this could also be useful for the office job etc for handling promotions?), and one that checks it: eg setJob(JobString) and hasJob(JobString). Perhaps even for NPCs too?

**Comment by vinfamy** [comment]
Yeah, I'll implement a more flexible and moddable system for jobs with v1.10 . TBH, when I first started the project, because I took inspiration from NewLife, in my head I didn't really think about anything more than a generic office job. That's why it's so difficult to mod in a new career path.

**Comment by vinfamy** [comment]
Rework and improve the system for jobs in the game to allow for more variety and complexity
Expand the office career path using the new job system
